src/runtime_src/core/tools/xbmgmt2/SubCmdConfigure.cpp

```diff
@@ -43,6 +43,22 @@ enum class config_type {
     reset
 };
 
+static
+std::string
+config_to_string(config_type value)
```

Comment:
While this code is indeed functionality, it will have issues scaling.  For example, if there is a new enumerated config type, it will:
1. Create a new empty entry into the map.
2. Return back that empty string name.

Instead, this code should be updated to help the next developer discover that the need to update this collection.  

Some ideas to think about:
1. One might original think that making the std::map const would address the issue, it will not and the build will fail for [] cannot be const for the reason mentioned earlier.
2. Make the collection a vector of pairs and create a lamda to search for the key, if not found  then throw an exception.
3. Create a switch statement.  The compilers are smart enought to recongize when a enumeration is missing and will produce an compile error. 

Each solutions has its own pros and cons.  I will let you decide which to choose.

Reply:
Makes sense to me. I was looking up some good ways to do this but the best choice was not obvious, thank you for giving me a write-up about which ones are better/worse.
I went with a switch case as I prefer the simpler syntax